core/src/main/resources/default-main.conf

```diff
@@ -15,7 +15,7 @@ remote.ssl.required = true
 ######################################################################
 
 # Regexes of allowed NON-SSL remote hosts separated by ","
-remote.nossl.hosts = localhost,*.apache.org,127.0.0.1
+remote.nossl.hosts = localhost,127.0.0.1,*.redhat.com
```

Comment:
let's leave this out; we'll configure it for our internal services separately, but not in the main codebase.

Reply:
should i comment it out completely - without localhost and 127.0.0.1 ? 
or just "*.redhat.com" value?

Reply:
I think I'd leave localhost and 127.0.0.1. I'm tempted to say 172.16 as well, but if it's on a public OpenShift cluster that could cause problems I guess.